src/integrated/soil_canopy_model.jl

```diff
@@ -285,50 +285,6 @@ function make_update_boundary_fluxes(
     return update_boundary_fluxes!
 end
 
-"""
```

Comment:
moved to separate file since this is used by multiple integrated models now, does not "belong" to this one.